README.md

```diff
@@ -5,34 +5,60 @@
 
 # Device Modeling Language
 
-DML is a language for writing device models for computer architecture
-simulators. Currently, the compiler only supports building models for
-the
-[Simics®](https://www.intel.com/content/www/us/en/download/645996/simics-simulator-public-release-preview.html)
-simulator, but other back-ends may be added in the future.
+The Device Modeling Language (DML) is a domain-specific language for
+writing fast functional or transaction-level device models for virtual
+platforms.  DML provides high-level abstractions suitable for
+functional device models, including constructs like register banks,
+registers, bit fields, event posting, interfaces between models, and
+logging. DML code is compiled by the DML Compiler (DMLC), producing C
+code with API calls tailored for a particular simulator.
 
-## Building and testing
+Currently, the compiler supports building models for the Intel®
+[Simics®](https://www.intel.com/content/www/us/en/developer/articles/tool/simics-simulator.html)
+simulator, but other back-ends may be added in the
+future.
+
+
+## Building and testing DMLC 
+
+To build the DML compiler (DMLC), you need to have a Simics
```

Comment:
You already said (DMLC) in the intro section, do we need to repeat it?

Reply:
Not really.